src/MICore/Debugger.cs

```diff
@@ -494,37 +492,13 @@ protected bool IsCoreDump
             {
                 await CmdBreak();
             }
-            CmdExitAsync();
+            await CmdAsync("-target-detach", ResultClass.done);
 
             return new Results(ResultClass.done);
         }
 
         public Task CmdBreakInternal()
         {
-            //TODO May need to fix attach on windows and osx.
-            if (IsLocalGdbAttach() && RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
```

Comment:
If you haven't already done so, please double check with @jacdavis what scenarios this was required for.